packages/core/src/StateNode.ts

```diff
@@ -289,7 +289,9 @@ class StateNode<
     /**
      * The initial extended state
      */
-    public context: Readonly<TContext> = undefined as any // TODO: this is unsafe, but we're removing it in v5 anyway
```

Comment:
This was public property - hard to tell if somebody was using it but since `StateNode` is also used for the "root" node it's not that unlikely that this was accessed by somebody at runtime.